test_that("Errors out on bad file", {
  expect_equal(2 * 2, 4)
})
